package.json

```diff
@@ -146,6 +146,18 @@
           "default": "haskell-language-server",
           "description": "Which language server to use."
         },
+        "haskell.languageServerReleasesPath": {
+          "scope": "resource",
+          "type": "string",
+          "default": "https://api.github.com/repos/haskell/haskell-language-server/releases",
+          "description": "a path to a json file"
+        },
+        "haskell.languageServerDownloadProxy": {
+          "scope": "resource",
+          "type": "string",
+          "default": "",
+          "description": "http:// https:// socks:// socks5:// pac+http://"
```

Comment:
If we are using the vscode http proxy setting, do we need to have our own proxy configuration in this extension too? I think we can just remove this one then

Reply:
a little bit too much, please don't pick the proxy part

Reply:
I create #313 for just Mirror Support part